libsolidity/ast/ASTAnnotations.h

```diff
@@ -166,9 +166,6 @@ struct UserDefinedTypeNameAnnotation: TypeNameAnnotation
 
 struct VariableDeclarationStatementAnnotation: StatementAnnotation
 {
-	/// Information about which component of the value is assigned to which variable.
```

Comment:
Can you please delete the whole struct?

Reply:
Yes.

Reply:
Ah no wait a second. The others inherit from `DocumentedAnnotation`, so they are not really "empty".

Reply:
Ok, they are still used in DocStringAnalyzer. I'll just remove this one here then.